### DISCUSSION TITLE
Multiple devices

### DISCUSSION THREAD

**Discussion by lamboid** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#344] [Q&A] [answered]
Multi floored house so we have a couple (at the moment) scooting around in the morning!
Is it possible to use the extractor on multiple devices?

When I tried adding a second "-platform xiaomi_cloud_map_extractor" hass just reports 

`ERROR (MainThread) [homeassistant.components.camera] Entity id already exists - ignoring: camera.xiaomi_cloud_map_extractor`

**Comment by lamboid** [comment, marked as answer]
just found #119 .
going to test now

**Comment by lisenet** [comment]
Why did I get this SPAM email?

![image](https://user-images.githubusercontent.com/8998418/209958680-c6d2e910-90e7-48b3-9468-1a653af34784.png)

**Reply by PiotrMachowski** [reply]
This is not my fault, there seems to be a wave of suspicious accounts posting suspicious comments under my repositories. They have been reported and removed, but sent emails can't be undone :(

**Comment by lisenet** [comment]
Are you 100% confident your system hasn't been pwned?

**Reply by PiotrMachowski** [reply]
Yes, it was a comment under this discussion
